src/core_plugins/kibana/public/dashboard/filter_utils.js

```diff
@@ -14,7 +15,7 @@ export class FilterUtils {
    * (e.g. goes in the query input bar), false otherwise (e.g. is in the filter bar).
    */
   static isQueryFilter(filter) {
-    return filter.query && filter.query.query_string && !filter.meta;
+    return filter.query && !filter.meta;
```

Comment:
Was this change to fix the separate issue where query DSL queries weren't placed back in the query bar correctly?